fcade-quark.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
This desktop file isn't provided properly by upstream, so I've made my own. Their `fcade.sh` wrapper script actually builds it inside the script but that is not viable for us since their wrapper does some other checks (specifically for `/usr/bin/wine`) that would cause us to fail out. I've contacted upstream about respecting the `PATH` and it's on their ToDo list, but for now this works.

Reply:
Snipper from upstream's `fcade.sh`:

```
if [ -x /usr/bin/xdg-mime ]; then
        # register fcade:// url handler
        mkdir -p ~/.local/share/applications/
        echo "[Desktop Entry]
Type=Application
Encoding=UTF-8
Name=FightCade Replay
Exec=${THIS_SCRIPT_DIR}/fcade.sh %U
Terminal=false
MimeType=x-scheme-handler/fcade
" > ~/.local/share/applications/fcade-quark.desktop
        xdg-mime default fcade-quark.desktop x-scheme-handler/fcade
fi
if [ -x /usr/bin/gconftool-2 ]; then
        gconftool-2 -t string -s /desktop/gnome/url-handlers/fcade/command "${THIS_SCRIPT_DIR}/fcade.sh %s"
        gconftool-2 -s /desktop/gnome/url-handlers/fcade/needs_terminal false -t bool
        gconftool-2 -t bool -s /desktop/gnome/url-handlers/fcade/enabled true
fi
```